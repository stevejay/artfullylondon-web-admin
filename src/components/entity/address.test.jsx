import React from 'react'
import { shallow } from 'enzyme'

import EntityAddress from '_src/components/entity/address'

it('should render correctly', () => {
  const wrapper = shallow(<EntityAddress fullAddress='26 Foo Street, Bar' />)
  expect(wrapper).toMatchSnapshot()
})
