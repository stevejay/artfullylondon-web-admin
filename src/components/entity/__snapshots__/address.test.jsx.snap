// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<React.Fragment>
  <AdditionalDetailHeading>
    Address
  </AdditionalDetailHeading>
  <AdditionalDetailContent>
    26 Foo Street, Bar
  </AdditionalDetailContent>
</React.Fragment>
`;
